lib/templates.js

```diff
@@ -96,6 +113,23 @@ function _getTitleFromServiceName(srv) {
     }
 }
 
+
+/**
+ * This is a function to get the version of the entity,
+ * validate it and log if it is not a valid semantical version.
+ *
+ * @param {object} entity An entity object.
+ * @returns Version of the entity with '1.0.0' as fallback value.
+ */
+function _getEntityVersion(entity) {
+    const entityVersion = entity.ordId.split(":").pop();
+    const version =  entityVersion.replace("v", "") + ".0.0"; // TODO: version can be stated/overwritten by annotation
+    if (!SEM_VERSION_REGEX.test(version)) {
+        Logger.warn(`Entity version "${version}" is not a valid semantic version.`);
+    }
+    return version;
```

Comment:
The default version is not an option. Maybe we should throw an exception in case of non-valid version?

Reply:
I think so, at least we should not provide invalid value. However, I assume this case is really rare in production environment, since we have validator protect us.

Reply:
If you look at the line above the `if (!SEM_VER....`)) {`, you can the comment: 
`// TODO: version can be stated/overwritten by annotation`
Simon told me to keep it that way until we clarify how to should be version set by the model/app-owners.
To his opinion, the right way would be to use an annotation, but told me to keep it this way for now.
Therefore, I would keep the logger(and not throwing the exception) for now until we have the final statement about the version.